api/Location.json

```
@@ -582,7 +582,8 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true
+              "version_added": true,
+              "notes": "Calling location.reload() inside a nested cross-origin frame reloads the top frame, if the top frame address was added to Trusted websites."
```

Hi @muodov and welcome to BCD! Thank you for opening this PR. This note was a bit unclear to me but I also couldn't test it myself (no Windows machine to test with). Here's how I understand your note:

If a page that contains a cross-origin iframe is added to _Trusted Sites_ and `reload()` is called from within the iframe, then the enclosing page will reload (not the page in the iframe, as you'd expect).

Do I understand that correctly?

---

Thanks @ddbeck :)

You are correct except for one thing: it is the address of the _top_ frame that should be Trusted.
So on the test page above, `https://x.zok.pw` must be added to Trusted, and `https://y.zok.pw` (the nested address) will call `location.reload()`.
In this case, the top frame gets relocated to `y.zok.pw`.

I have no explanation to why this could make sense, but it's also reproducible in (pre-Chromium) Edge.

---

Interesting (and weird)! I'm going to suggest we be rather verbose here because it is complicated. It seems to me that the key thing is the trusted sites aspect, so I want to emphasize that. What do you think of this wording?

```suggestion
              "notes": "If a page added to <em>Trusted Sites</em> contains a cross-origin iframe, then calling <code>reload()</code> from within the iframe reloads the trusted page (in other words, the top page reloads, not the iframe)."
```

Is that an improvement? Or have I muddied things?

---

Yeah, I like this better 👍